plugins/woocommerce-admin/client/customize-store/assembler-hub/auto-block-preview.tsx

```diff
@@ -241,7 +242,11 @@ function ScaledBlockPreview( {
 						// @ts-ignore disabled prop exists
 						scrolling={ isScrollable ? 'yes' : 'no' }
 						tabIndex={ -1 }
-						readonly={ false }
+						readonly={
+							isFullComposabilityFeatureAndAPIAvailable()
+								? false
+								: true
+						}
```

Comment:
Nitpick: Maybe we can simplify this to following
```suggestion
						readonly={
							! isFullComposabilityFeatureAndAPIAvailable()
						}
```

Reply:
Fixed with [20d2870](https://github.com/woocommerce/woocommerce/pull/48752/commits/20d28707a7a8f3b15d71c26d17a5973f8033881e)